workshops/Minecraft_Modding/README.md

```diff
@@ -0,0 +1,178 @@
+
+``` 
+{ 
+  "parent": "item/generated", 
+  "textures": { 
+    "layer0": "modname:items/item" 
+  } 
+} 
+```
+Remember to replace modname and item of course. 
+
+Now we will be putting the item's image in. I will provide an image, but you can make your own in paint, just make sure the image size is 16x16 pixels. The file must have the same itemname and be in .png. Here is the example item I will be using. 
+
+![](https://cloud-qpr3fyvd6.vercel.app/myitem.png). 
+
+Place the png file in the `textures/items` directory. 
+
+Now run Minecraft using Shift-F10 or the triangle play button and see your item in Minecraft! 
+
+# Part 5: Making your first block! 
+
+Making your first block is actually very simple now that we have most of the files and registries made.  
+
+To get started go back to your `src/main/java/com/username/modname/util` folder and open the RegistryHandler file. Inside of that we are going to just use the same methods from before but change them up a little. 
+
+Duplicate the DeferredRegister line at the start and change the `<Item>` to `<Block>` and rename `ITEMS` to `BLOCKS` in both cases. Now, duplicate the `ITEMS.register` line and change `ITEMS` to `BLOCKS`. Now duplicate the `RegistryObject<Item>` line and change Item to, you guessed it, Block. Now create a name for the variable, I am calling mine MYBLOCK. Now change the `ITEMS` part to `BLOCKS` and put a lowercase version of your block name. Also change Itembase to the `MYBLOCK::new`. 
+
+One more thing, make another RegistryObject like last time, but keep it as an `<Item>`. Change the name to something like MY_BLOCK_ITEM. Then make it equal to `ITEMS.register("myblock", () -> new BlockItemBase(MY_BLOCK.get()));`. 
+
+Now within your modname directory, make a new directory called blocks. Now make two classes, one called BlockItemBase, and another with your block name. Within your BlockItemBase just use this snippet as we are just adding it to our mod's tab. 
+
+```java 
+public class BlockItemBase extends BlockItem { 
+    public BlockItemBase(Block block) { 
+        super(block, new Item.Properties().group(kaimod.TAB)); 
+    } 
+} 
+```
+Change kaimod to the name of your mod. 
+
+Now open your block class and this is where we set the properties of the item! At the end of the class name, make sure it extends Block. Now make a method inside of your class with the same name, make it public as well. Within the method write 
+``` java
+super(Block.Properties.create() 
+    ); 
+```
+Within the parenthesis write Material. then your IDE should list all the Materials in the game. This just means what type of block it is. I set mine to `Material.IRON`. Now after the parenthesis put down a period and there should be a lot of different methods available. For the sake of simplicity, I will just make it only minable with a pickaxe. You can do that by setting `.harvestTool(ToolType.Pickaxe)`.  

```

Comment:
Which parenthesis? The ones on `create()`? Make clearer. Also break down this paragraph and include a gif of IntelliJ's autocompletion so that people better understand what to expect.